node/light.go

```diff
@@ -118,6 +120,24 @@ func (ln *LightNode) OnStart() error {
 	}
 
 	ln.fraudService = ln.proofServiceFactory.CreateProofService()
+	if err := ln.fraudService.AddVerifier(types.StateFraudProofType, func(fraudProof fraud.Proof) (bool, error) {
+		stateFraudProof, ok := fraudProof.(*types.StateFraudProof)
+		if !ok {
+			return false, errors.New("unknown fraud proof")
+		}
+		resp, err := ln.proxyApp.Consensus().VerifyFraudProofSync(
+			abci.RequestVerifyFraudProof{
+				FraudProof:           &stateFraudProof.FraudProof,
+				ExpectedValidAppHash: stateFraudProof.ExpectedValidAppHash,
+			},
+		)
+		if err != nil {
+			return false, err
+		}
```

Comment:
Can we reuse this current lambda function between `full.go` and `light.go` to make the code more DRY?

Reply:
done.